ui/src/app/job-details/panels/resources-table/resources-table.component.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  Component,
+  Input,
+  OnChanges,
+  SimpleChanges
+} from '@angular/core';
+import {ResourceUtils} from '../../../shared/utils/resource-utils';
+
+
+@Component({
+  selector: 'jm-resources-table',
+  templateUrl: './resources-table.component.html',
+  styleUrls: ['./resources-table.component.css'],
+})
+export class ResourcesTableComponent implements OnChanges {
+  readonly resourcesColumns: string[] = ["key", "value"];
```

Comment:
This will produce a table with headers of "key" and "value" right? If so may be worth going over with Marty whether we should try and just hide that entirely.

Reply:
It actually does not since we don't have a `mat-header-row` in the template using those columns. I thought it looked pretty bad with those headers. 

However,  this is basically unnecessary it can just be inlined into the template since its used once, so removed.